Add NoArm64 testRequires and skip Schema1RegistrySuite on arm64

coolljt0725 opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/docker/docker/blob/master/CONTRIBUTING.md

** Make sure all your commits include a signature generated with `git commit -s` **

For additional information on our contributing process, read our contributing
guide https://docs.docker.com/opensource/code/

If this is a bug fix, make sure your description includes "fixes #xxxx", or
"closes #xxxx"

Please provide the following information:
-->

**\- What I did**

**\- How I did it**

**\- How to verify it**

**\- Description for the changelog**

<!--
Write a short (one line) summary that describes the changes in this
pull request for inclusion in the changelog:
-->

**\- A picture of a cute animal (not mandatory but encouraged)**

schema1 manifests is not working on ARM64, we should skip integration-cli
tests for schema1 manifests on ARM64.
https://github.com/docker/docker/blob/master/Dockerfile.aarch64#L108

Signed-off-by: Lei Jitang leijitang@huawei.com

justincormack commented:
LGTM

justincormack commented:
Should any of the other tests that are marked as not ppc64le etc also be marked as not arm64?

coolljt0725 commented:
@justincormack  I haven't run the integration test with the master on arm64, we run the integration test with 1.11.1 and we only found `SetUpTest` and `TestRunSeccompProfileDenyChmod` failed. those tests marked as not ppc64le also passed. We will run the integration test with master on arm64 soon and we may find more issues.

justincormack commented:
Ok, well we can add them later if they turn up. I may be able to do some testing too, I did some light testing a few weeks back.

coolljt0725 committed:
Add NoArm64 testRequires and skip Schema1RegistrySuite on arm64

schema1 manifests is not working on ARM64, we should skip integration-cli
tests for schema1 manifests on ARM64.

Signed-off-by: Lei Jitang <leijitang@huawei.com>

runcom commented:
LGTM

runcom commented:
ping @vdemeester
